.gitignore

```diff
@@ -383,12 +383,27 @@ Version.cpp
 Reader*Test.data
 /pwiz_aux/msrc/utility/vendor_api/*/EULA.*
 /pwiz_aux/msrc/utility/vendor_api/ABI/LicenseKey.h
+/pwiz_aux/msrc/utility/vendor_api/ABI/*.dll
+/pwiz_aux/msrc/utility/vendor_api/Agilent/x64/
+/pwiz_aux/msrc/utility/vendor_api/Agilent/x86/
+/pwiz_aux/msrc/utility/vendor_api/Bruker/x64/
+/pwiz_aux/msrc/utility/vendor_api/Bruker/x86/
 /pwiz_aux/msrc/utility/vendor_api/Bruker/**/*.h
```

Comment:
Would /pwiz_aux/msrc/utility/vendor_api/**/*.dll cover all these?

Reply:
Yes I can change that

Reply:
Actually, this would cut out some versioned files like the /pwiz_aux/msrc/utility/vendor_api/ABI/x64/SQLite_v1.0.98/System.Data.SQLite.DLL

Two alternatives:
Exclude the entire /pwiz_aux/msrc/utility/vendor_api/ directory with `/pwiz_aux/msrc/utility/vendor_api/**/*` and then either:
1) make future additions with `git add --force` (and maybe add a comment in the .gitignore indicating this is the intent), or
2) add each of our included files as explicit exceptions, e.g. 
```
!/pwiz_aux/msrc/utility/vendor_api/**/*.jam
!/pwiz_aux/msrc/utility/vendor_api/ABI/WiffFile.cpp
!/pwiz_aux/msrc/utility/vendor_api/ABI/WiffFile.hpp
...
```

Reply:
Yeah let's just add the versioned DLLs as explicit exceptions. That damn SQLite DLL, I want to kill it!

Reply:
My mistake, adding /pwiz_aux/msrc/utility/vendor_api/**/*.dll won't cover those four directories because they also contain some .dll.config, .xml, and .lib files as well. So I'm not really seeing a clean way to exclude them without just excluding those four directories.

Reply:
These files are checked into git, right? So why does it matter whether they're included by an ignore rule? If we need to add a new DLL to one of these vendors that have non-private DLLs, it's true the ignore rule would make it harder to notice that it hasn't been added yet, but using exceptions wouldn't help that either. Only individually naming all the excluded DLLs would really solve that, right? It's not worth it. We'll just force add them on a case-by-case basis.

Reply:
Okay that makes sense. It didn't seem worth managing all of that. That's why I was leaning towards treating all of `/pwiz_aux/msrc/utility/vendor_api/` as something of a build output directory, the contents of which we don't really control, and then resort to `git add --force` for any changes to the vendor api.

Reply:
Getting back to the original problem, here are the non-dll files that I was needing to exclude in order to reduce clutter on calls to `git status`:

```
pwiz_aux/msrc/utility/vendor_api/Agilent/x64/BaseDataAccess.dll.config
pwiz_aux/msrc/utility/vendor_api/Agilent/x86/BaseDataAccess.dll.config
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/BDal.BCO.Constants.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/BDal.BCO.Objects.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/BDal.CCO.Constants.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/BDal.CCO.Objects.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/FlexVariableTable.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/ICRVariableTable.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/Interop.EDAL.SxS.manifest
pwiz_aux/msrc/utility/vendor_api/Bruker/x64/Interop.HSREADWRITELib.SxS.manifest
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/BDal.BCO.Constants.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/BDal.BCO.Objects.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/BDal.CCO.Constants.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/BDal.CCO.Objects.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/FlexVariableTable.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/ICRVariableTable.xml
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/Interop.EDAL.SxS.manifest
pwiz_aux/msrc/utility/vendor_api/Bruker/x86/Interop.HSREADWRITELib.SxS.manifest
```
The simplest pattern I saw in these was just
```
/pwiz_aux/msrc/utility/vendor_api/Agilent/x64/
/pwiz_aux/msrc/utility/vendor_api/Agilent/x86/
/pwiz_aux/msrc/utility/vendor_api/Bruker/x64/
/pwiz_aux/msrc/utility/vendor_api/Bruker/x86/
```

Maybe that's still the cleanest? It's my fault for adding confusion by at first saying that a *.dll exclusion would cover these all.

Reply:
Yeah. Go for that (in addition to the *.dll one)

Reply:
Will do 👍

Reply:
Done in https://github.com/ProteoWizard/pwiz/pull/332/commits/a43b259829ab79b295536f1bac27d6964dfc42b3